app/config/sui/contracts-testnet.json

```diff
@@ -0,0 +1,9 @@
+{
+	"nbtcOTC": {
+		"packageId": "0x4995e309e990a6a93224153108b26bf79197b234c51db6447bbae10b431c42fb",
+		"module": "nbtc_swap",
+		"swapFunction": "swap_sui_for_nbtc",
+		"vaultId": "0xf280477ca196a4bced5e1db4cd82fcdd647b55585b1d3838dcd8e1b829d263a4",
+		"pricePerNBTCInSUI": 25000
```

Comment:
again, the `pricePerNBTCInSUI is not a part of the network config, i am not sure if we should mix it with the package id etc.

Reply:
It's a fixed price. Let's keep it for now. We can remove it later. What say?

Reply:
then lets move it to a constant not config